build.py

```diff
@@ -152,6 +136,10 @@ def isExceptionTopIndex(self, attr):
 
 # ============================================================================
 
+cwd = str(os.getcwd())
+lsResults = os.listdir(cwd)
+
+isMkdocsInstalled = subprocess.check_output(['mkdocs', '--version']).startswith(b'mkdocs')
```

Comment:
These declarations originally were right here,   
but I moved these variables up to the top in `#19f5183` to improve readability.
(There were too many important variables that were declared inside of this code block.)

Considering how we will probably divide `PostMkdocsParser` into a separate file, I think this is a moot point; they will be located in about the same area, at the top where variables should be declared. LGTM.